Packs/Smokescreen_IllusionBLACK/Integrations/Smokescreen_IllusionBLACK/Smokescreen_IllusionBLACK.yml

```diff
@@ -84,6 +88,10 @@ script:
     description: Check if Active Directory user is a decoy
     execution: false
     name: illusionblack-is-user-decoy
+    outputs:
+    - contextPath: IllusionBlack.IsUserDecoy
+      description: A mapping of user and whether it's an IllusionBLACK AD user decoy
+      type: Unknown
```

Comment:
```suggestion
      type: Boolean
```

Reply:
This is not boolean, it is a dict:
```
{
  "HOST": true/false
}
```

Reply:
Added context output:

- IllusionBlack.IsUserDecoy
- IllusionBlack.IsUserDecoy.User
- IllusionBlack.IsUserDecoy.Value

```
{
    "IllusionBlack": {
        "IsUserDecoy": {
            "User": "sqladmin",
            "Value": true
        }
    }
}
```